### PR TITLE
Lu 26/dialouge

### PR DESCRIPTION
**Pull request by Aman9804**
Please review and approve

### CODE REVIEW

**Review by sanketXynes — APPROVED** [review]
Approved

File: src/components/Dialouge/dialouge.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      secondaryBtnOnPress,
+      secondaryIcon,
+      secondaryBtnlabel,
+      visible = false,
+      closeIcon = false,
+      ...rest
+    },
+    ref
+  ) => {
+    const StyledDialouge = styled(BasicStyledDialouge)`
```
Comment (archan96) [inline src/components/Dialouge/dialouge.tsx]: Move these three to styles.ts

File: src/components/Dialouge/dialouge.type.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { AssetProps } from "@icons/types/asset.types";
+import React from "react";
+
+export interface DialougeProps extends React.HtmlHTMLAttributes<HTMLDivElement> {
+    visible: boolean;
```
Comment (archan96) [inline src/components/Dialouge/dialouge.type.tsx]: Make this optional

**Review by archan96 — CHANGES_REQUESTED** [review]
Make the following changes

**Comment by Aman9804** [comment]
Changes made

File: src/components/Dialouge/dialouge.type.tsx (hunk 2 above)
Comment (Aman9804) [reply src/components/Dialouge/dialouge.type.tsx]: Done

File: src/components/Dialouge/dialouge.tsx (hunk 1 above)
Comment (Aman9804) [reply src/components/Dialouge/dialouge.tsx]: Done

**Review by archan96 — APPROVED** [review]
Sounds good

**Review by archan96 — APPROVED** [review]
All things looks good

**Review by xynes-admin — APPROVED** [review]
All things are fine

**Review by sanketXynes — APPROVED** [review]
Approved

**Pull request merged**